opba-protocols/hbci-protocol/src/main/java/de/adorsys/opba/protocol/hbci/service/consent/HbciReadAccountListFromCache.java

```diff
@@ -28,6 +28,10 @@ private void convertConsentToResponseIfPresent(DelegateExecution execution) {
             if (null != ctx.getCachedResult()) {
                 ctx.setResponse(ctx.getCachedResult().getAccounts());
             }
+
+            if (ctx.getUseCache() == Boolean.FALSE) {
```

Comment:
This will cause the transactions cache to be removed, you need only update accounts and keep transactions